index.js

```diff
@@ -0,0 +1,513 @@
+	message.delete(); // Deleting the original message - this isn't strictly needed
+	let author = message.author.username;
+	let member = message.mentions.members.first();
+	let outputExtra = ' to ';
+
+	if (!member && input[1] !== null) {
+		for (let i = 1; i < input.length; i++) {
+			outputExtra += input[i];
+		}
+	}
+	else if (!member) {
+		outputExtra = "";
+	}
+	else outputExtra += member;
+
+	message.channel.send(author + ' has paid respects' + outputExtra, {
+	  embed: {
+		  color: 1752220,
+			description: "",
+			image: {
+				url: fGifs[0]
+			}
+		}
+	});
+}
+
+/**
+ * @timezones
+ * @param message
+ */
+function timezones(message) {
+	let adelaide = new Date().toLocaleString("en-US", {timeZone: "Australia/Adelaide"});
+	let usEast = new Date().toLocaleString("en-US", {timeZone: "America/New_York"});
+	let usWest = new Date().toLocaleString("en-US", {timeZone: "America/Los_Angeles"});
+	let ukLon = new Date().toLocaleString("en-US", {timeZone: "Europe/London"});
+
+	adelaide = new Date(adelaide);
+	usEast = new Date(usEast);
+	usWest = new Date(usWest);
+	ukLon = new Date(ukLon);
+
+	message.channel.send("```AUS Central: "+adelaide.toLocaleString()+"\nUS East: "+usEast.toLocaleString()+"\nUS West: "+usWest.toLocaleString()+"\nUK: "+ukLon.toLocaleString()+"```");
+}
+
+/**
+ * @embedAvatarIcon
+ * @param message
+ */
+function embedAvatarIcon(message) {
+	let user = message.mentions.users.first();
+	let embed;
+
+	if(user == null) {
+		embed = new Discord.RichEmbed()
+			.setImage(message.author.displayAvatarURL)
+			.setColor('#275BF0');
+	}
+	else {
+		embed = new Discord.RichEmbed()
+			.setImage(user.avatarURL())
+			.setColor('#275BF0');
+	}
+
+	message.channel.send(embed);
+}
+
+/**
+ * @embedServerIcon
+ * @param message
+ */
+function embedServerIcon(message) {
+	let embed = new Discord.RichEmbed()
+		.setImage(client.guilds.get(message.guild.id).iconURL)
+		.setColor('#275BF0');
+
+	message.channel.send(embed);
+}
+
+/**
+ * @serverInformation
+ * @param message
+ */
+function serverInformation(message) {
+	let embed = new Discord.RichEmbed()
+		.setColor('#275BF0')
+		.addField("Server Creation: ", client.guilds.get(message.guild.id).createdAt);
+
+	message.channel.send(embed);
+}
+
+/**
+ * @userInformation
+ * @param message
+ */
+function userInformation(message) {
+	let user = message.mentions.users.first();
+	if (!user) {
+		user = message.author;
+	}
+
+	let embed = new Discord.RichEmbed()
+		.setColor('#275BF0')
+		.addField("Name: ",`${user.username}#${user.discriminator}`);
+	user = message.guild.member(user); // Believe this needs to be called here because the username and discriminator cannot be called from guild.member
+	embed.addField("Server Join Date: ", `${user.joinedAt}`)
+		.addField("Status: ", `${user.presence.status}`);
+
+	message.channel.send(embed);
+}
+
+/**
+ * @messageUser
+ * @param message
+ */
+function messageUser(message) {
+	if(message.author.id === author_id){
```

Comment:
maybe change `author_id` to `bot_owner_id`, was a bit misleading to read at first

Reply:
Done, will update that in a later PR